src/App.php

```diff
@@ -234,52 +236,53 @@ public function callExit($from_shutdown = false)
      * @param Throwable $exception
      *
      * @throws \atk4\core\Exception
+     * @throws ExitApplicationException
      *
      * @return bool
      */
     protected function caughtException(Throwable $exception)
     {
         $this->catch_runaway_callbacks = false;
 
-        // Use new App() instead of static() to prevent broken exception
-        // message display due to conflict with existing layout
-        $l = new self();
-        $l->title = 'L'.$exception->getLine().': '.$exception->getMessage();
-        $l->catch_runaway_callbacks = false; // Allow exceptions within modals
-        $l->initLayout('Centered');
+        // just replace layout to avoid any extended App->_construct problems
+        // it will maintain everything as in the original app StickyGet, logger, Events
+        $this->html = null;
```

Comment:
in place of creating a new app for exception, why not destroy the render tree making $app->html = null
and reuse the actual app, all $defaults will be respected.